PublisherView.js

```diff
@@ -81,6 +81,12 @@ class PublisherView extends React.Component {
   sayHello() {
     OpenTokPublisherViewManager.sayHello();
   }
```

Comment:
yeah - we should! it does work (it's been merged on the native lib side).. was mainly used for testing/debugging all the native bridge.
I'll remove this part now